app/views/welcome/index.html.erb

```diff
@@ -1,12 +1,9 @@
 <% content_for :head do -%>
   <%= render 'welcome/meta_tags' -%>
 <% end -%>
-<div class="redtop home">
+<div class="home">
   <div class="container">
     <h1><%= t :front_page_header_html -%></h1>
-    <% unless user_signed_in? -%>
-      <%= render 'subscriptions/form' -%>
-    <% end -%>
```

Comment:
Are we sure this is going away for anonymous users?

Reply:
My bad, restoring.